kolibri/plugins/learn/assets/src/views/search-box/index.vue

```diff
@@ -132,10 +133,15 @@
     &::placeholder
       color: $core-text-annotation
 
+    &::-ms-clear
+      display: none
```

Comment:
mind adding a code comment explaining what this does?

Reply:
Comment added :+1: